src/openzaak/components/catalogi/openapi.yaml

```diff
@@ -8186,7 +8186,7 @@ components:
           type: string
           maxLength: 1000
         zaaktype:
-          title: Is van
+          title: Zaaktype
```

Comment:
this shouldn't happen, right?

Reply:
Oh, I thought there weren't any changes. I'll fix.

Reply:
I'm keeping it this way. We'll solve title in another way and I don't find it important enough.